### PR TITLE
fixed dropdown issue

### LINKED ISSUE #1247: Different change state dropdown for admin/events#index and admin/events#show
In admin/events#index in the **State** column if an event is **withdrawn** it cannot be changed to new (There is no drop down and thus no option to _Start Review_).

In admin/events/#show that's no longer the case and an event which is in state **withdrawn** can change to state **new**

According to https://github.com/openSUSE/osem/blob/master/app/models/event.rb#L54 it should be possible to go from **Withdrawn** --> to **New**
Isn't that so @ChrisBr ?

### PR DESCRIPTION
**Pull request by siddhantbajaj**
Fix #1247

### CODE REVIEW

**Comment by ChrisBr** [comment]
@siddhantbajaj please check why tests are failing. Otherwise LGTM 👍

**Comment by differentreality** [comment]
@ChrisBr tests fail sometimes due to https://github.com/openSUSE/osem/pull/1281 (check my last comment) which is being fixed in #1287

File: app/views/admin/events/index.html.haml [hunk 1]
```diff
@@ -176,14 +176,11 @@
...
-                Withdrawn
-              - else
-                .btn-group
-                  %button{:type=>"button", :class=>"btn btn-link dropdown-toggle", "data-toggle"=>"dropdown"}
-                    = event.state.humanize
-                    %span.caret
-                  %ul.dropdown-menu{:role=>"menu"}
-                    = render 'change_state_dropdown', event: event
+              .btn-group
+                %button{:type=>"button", :class=>"btn btn-link dropdown-toggle", "data-toggle"=>"dropdown"}
```
Comment (differentreality) [inline app/views/admin/events/index.html.haml]: Would be great if you can apply proper styling as well, ie new ruby syntax and single quotes.

Comment (siddhantbajaj) [inline app/views/admin/events/index.html.haml]: done.

**Comment by Ana06** [comment]
LGTM  @siddhantbajaj please squash the last commit into the first one: https://ariejan.net/2011/07/05/git-squash-your-latests-commits-into-one

**Comment by siddhantbajaj** [comment]
done :+1:

**Comment by differentreality** [comment]
:+1:  :tada:  Thank you for the contribution @siddhantbajaj

**Pull request merged**